### PR TITLE
Replace time with std::time

### PR DESCRIPTION
**Pull request by serprex**
Also removed counters.rs.bk

### CODE REVIEW

**Comment by brayniac** [comment]
@serprex - thanks for the PR. There are a few issues we need to address, but before I can really show them, let me fix the benchmark that's included. Missed some s/// before I published.

**Comment by brayniac** [comment]
Okay - just merged #3 to fix the benchmark. Once you merge in those fixes, you should be able to run the benchmark per the following:

```shell
cd benches && cargo build --release && ./target/release/benches
```

You'll see output like this: (this is from master)
```
2016-07-21 00:13:20 INFO  [benchmark] tic benchmark
2016-07-21 00:13:20 INFO  [benchmark] producers: 1
2016-07-21 00:13:30 INFO  [benchmark] -----
2016-07-21 00:13:30 INFO  [benchmark] Window: 1
2016-07-21 00:13:30 INFO  [benchmark] Requests: Total: 65304855
2016-07-21 00:13:30 INFO  [benchmark] Responses: Ok: 65304855
2016-07-21 00:13:30 INFO  [benchmark] Rate: 7256095.00 rps
2016-07-21 00:13:30 INFO  [benchmark] Latency: min: 26 ns max: 0 ns
2016-07-21 00:13:30 INFO  [benchmark] Percentiles: p50: 33 ns p90: 37 ns p99: 50 ns p999: 92 ns p9999: 1382 ns
2016-07-21 00:13:40 INFO  [benchmark] -----
2016-07-21 00:13:40 INFO  [benchmark] Window: 2
2016-07-21 00:13:40 INFO  [benchmark] Requests: Total: 64162006
2016-07-21 00:13:40 INFO  [benchmark] Responses: Ok: 64162006
2016-07-21 00:13:40 INFO  [benchmark] Rate: 3376947.68 rps
2016-07-21 00:13:40 INFO  [benchmark] Latency: min: 26 ns max: 460456 ns
2016-07-21 00:13:40 INFO  [benchmark] Percentiles: p50: 33 ns p90: 37 ns p99: 48 ns p999: 87 ns p9999: 278 ns
```

File: src/lib.rs [hunk 1]
```diff
@@ -377,9 +377,9 @@ impl Receiver {
 
             try_handle_http(&server, &http_histogram, &gauges, &global_counters);
 
-            let t1 = time::precise_time_ns();
+            let t1 = Instant::now();
 
-            if t1 - t0 >= (duration as u64 * ONE_SECOND) {
+            if (t1 - t0).as_secs() >= (duration as u64 * ONE_SECOND) {
```
Comment (brayniac) [inline src/lib.rs]: One of this issues is here, this should be changed to: (as duration is already in seconds, this was seconds -> ns conversion)
```rust
if (t1 - t0).as_secs() >= (duration as u64) {
```

**Comment by brayniac** [comment]
That's looking better. Looks like there is a bug in calculating the rate for the first window (when duration = 1 instead of 10). But this appears to the the case for master as well.

Is there any chance we're losing resolution for calculating the rate itself? It seems that .as_secs() drops any fractional second component. That error could be quite high for 1s integration periods if the loop is delayed. Please correct me if I'm wrong here, I've mainly been working with time::precise_time_ns() for most purposes.

**Comment by serprex** [comment]
We aren't losing precision since you were using integer division before. If you'd like to keep the fractional component then a good helper function is `duration.as_secs() as f64 + duration.subsec_nanos() as f64 / 1000_000_000.0`

**Comment by brayniac** [comment]
Ah. Yeah, totally valid. Cool, I'm happy to merge this. Thanks again for the PR!

**Pull request merged**